libs/payments/paypal/src/lib/paypal.types.ts

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export enum BillingAgreementStatus {
+  Active = 'active',
+  Cancelled = 'cancelled',
+}
```

Comment:
I remember there being a discussion on a previous PR on enums vs Objects, and it was decided there to go with objects. Should the same be done here?

https://github.com/mozilla/fxa/pull/16314#discussion_r1475088723

Reply:
Possible discussions around this topic don't need to hold up the PR. Could always come back to it in the future.